### ISSUE TITLE
The task has been configured with an unsupported combination of account settings and run time options.

### ISSUE THREAD

**Issue by dahall** [dahall/TaskScheduler#176]
Hi,

Since I've updated to the latest stable release(2.5.4) I'm getting the following error:
_The task has been configured with an unsupported combination of account settings and run time options. (Exception from HRESULT: 0x80041314)_
This did not happen with version 2.4.2 and lower (doublechecked it)

What happens is:
I'm contacting a Win7 machine but i specify the taskservice to use 1.1 version.
Then i register a task with the serviceaccount.
I've been able to reproduce this error several times

Can you confirm this as a bug?

Code (C#):
```
using (TaskService ts = new TaskService(hostname, null, null, null, true))
{
   TaskDefinition tduser = ts.NewTask();
   bool newVer = (ts.HighestSupportedVersion >= new Version(1, 2));
   tduser.Actions.Add(new ExecAction("cmd", "Do something", null));
   tduser.Settings.DisallowStartIfOnBatteries = false;
   tduser.Settings.StopIfGoingOnBatteries = false;
   tduser.RegistrationInfo.Description = "My Task";
   if (newVer) //v2.0 
   {
      tduser.Principal.RunLevel = TaskRunLevel.Highest;
      tduser.Settings.AllowDemandStart = true;
   }
   ts.RootFolder.RegisterTaskDefinition("My Task", tduser, TaskCreation.CreateOrUpdate,
      "system", null, TaskLogonType.ServiceAccount); //this crashed
   ts.GetTask("My Task").Run();
}
```

Originally posted: 2015-11-25T01:42:40

**Comment by dahall** [comment]
Corrected issue. Look for fix in forthcoming 2.5.5 release.

Originally posted: 2015-11-28T20:38:18

**Comment by dahall** [comment]
Do you have an eta for that version?

Originally posted: 2015-12-02T01:43:44

**Comment by dahall** [comment]
This weekend most likely. I'm also working on another bug for that release.

Originally posted: 2015-12-02T08:35:30

**Comment by dahall** [comment]
Hi,

Thanks for the 2.5.5 update, however it does not seem to be working 100%.

Could you please verify
 - When i connect from my local computer to my local computer => it works
 - When i connect from my local computer to a computer in our domain => it fails with the following msg
   _The network path was not found. (Exception from HRESULT: 0x80070035)_

Part where it crashes
_using (TaskService ts = new TaskService(hostname, null, null, null, true))_

Originally posted: 2015-12-08T01:56:15

**Comment by dahall** [comment]
** Closed by dahall 12/7/2015 2:30 PM

Originally posted: 2015-12-08T01:57:01

**Comment by dahall** [comment]
Just to be clear, this was all working previous to 2.5.4 (2.4.x)? Connecting to Win7 via the v1 mode is much less reliable as it has to use a file share to access the tasks.

Originally posted: 2015-12-09T23:45:21

**Comment by dahall** [comment]
This was indeed working.

The reason why i know this is because we have a subnet of Win 7 computer that all return version 1.1 while they should have the newer version (i know, it's crazy...)

I used to be able to access the scheduler, but not anymore.


Originally posted: 2015-12-14T07:25:47

**Comment by dahall** [comment]
Please try it again now with released version 2.5.13. Also, I have posted here a diagnostic tool. Will you run it on both the systems (host and client) and let me know the results you get?

Originally posted: 2016-02-07T10:55:40

**Comment by dahall** [comment]
Originally posted: 2016-02-24T08:40:16
